oolab/src/main/java/agh/ics/oop/SimulationEngine.java

```diff
@@ -14,13 +18,14 @@ public SimulationEngine(MoveDirection[] moveDirection, IWorldMap map, Vector2d[]
             map.place(a);
             System.out.println(a.getPosition());
             a.addObserver((IPositionChangeObserver) map);
+            a.addObserver((IPositionChangeObserver) app);
```

Comment:
Dodanie observera ok, ale to powinno się wydarzyć w `App`, po to mamy observera żeby `SimulationEngine` nie musiał nic wiedzieć o istnieniu `App`.